src/views/Editor.vue

```diff
@@ -114,6 +147,9 @@ async function handleProposeClick() {
       </div>
     </nav>
     <Container v-if="proposals[proposalKey]" class="pt-5 s-box">
+      <UiAlert v-if="!fetchingVotingPower && !votingPowerValid" type="error">
+        You do not have enough Voting Power to create proposal in this space.
```

Comment:
We dont need the uppercase here for voting power
```suggestion
        You do not have enough voting power to create proposal in this space.
```